pages/compare.py

```diff
@@ -169,17 +134,15 @@ def list_models(
 myscopes_response = get_userapp_scopes(user_or_secrets_stub, userDataObject)
 validate_scopes(all_needed_scopes, myscopes_response.scopes)
 
-lister = ClarifaiResourceLister(
-    user_or_secrets_stub, user_or_secrets_auth.user_id, user_or_secrets_auth.app_id, page_size=16)
+#lister = ClarifaiResourceLister(
```

Comment:
Are this still needed ?, we can remove all commented code

Reply:
Removed this and all unused functions in the code.